deploy/jiva-csi-ubuntu-16.04.yaml

```diff
@@ -270,6 +270,11 @@ spec:
             - "--enableiscsidebug=true"
             # logging level for klog library used in k8s packages
             #- "--v=5"
+            # retrycount is the max number of retries per nodeStaging rpc
+            # request on a timeout of 5 sec
+            # This count has been set to 30 for sanity test cases as it takes
+            # time in minikube
```

Comment:
@kmova Can you please go through this comment.

Reply:
@payes it takes time since localpv provisioner spawns a initcontainer to provision the pvc, and replica stays in pending state until the initcontainer is finished doing stuff.